### PR TITLE
Delete whole Element by tap on Delete button

### PR DESCRIPTION
**Pull request by scipianne**

### CODE REVIEW

File: Keyboard/KeyboardViewController.swift [hunk 1]
```diff
@@ -207,3 +207,27 @@ extension KeyboardViewController: UITableViewDelegate {
...
+            let matches = regex.matches(in: context, options: [],
+                                        range: NSRange(location: 0,
+                                                       length: context.characters.count))
+            if let last = matches.last,
+                last.range.location + last.range.length == context.characters.count {
+                lengthOfSymbol = last.range.length
+            }
+        }
+        
+        for _ in 0..<lengthOfSymbol {
```
Comment (ogrenich) [inline Keyboard/KeyboardViewController.swift]: @scipianne Use `.forEach` operator in future for cases like this one ;-)

File: Keyboard/KeyboardViewController.swift [hunk 2]
```diff
@@ -207,3 +207,27 @@ extension KeyboardViewController: UITableViewDelegate {
     }
     
 }
+
+extension KeyboardViewController {
+    
+    func deleteLast() {
```
Comment (ogrenich) [inline Keyboard/KeyboardViewController.swift]: @scipianne Delete last Symbol or just character?

File: Keyboard/KeyboardViewController.swift (hunk 1 above)
Comment (scipianne) [reply Keyboard/KeyboardViewController.swift]: @ogrenich do you mean like
```
[0..<lengthOfSymbol].forEach { [weak self] _ in
    self?.textDocumentProxy.deleteBackward()
}
```

Comment (ogrenich) [reply Keyboard/KeyboardViewController.swift]: @scipianne Right, does it work so?

Comment (scipianne) [reply Keyboard/KeyboardViewController.swift]: @ogrenich now it does 😸

**Pull request merged**